args.py

```diff
@@ -0,0 +1,82 @@
+import json
+import PySimpleGUI as sg
+
+import argparse
+import purchase_invoice
+from api import Api
+from api_wrapper import api_wrapper_test
+from settings import VALIDITY_DATE
+from version import VERSION
+
+
+def arg_parser():
+    parser = argparse.ArgumentParser\
+              (description='ERPNext client für Solidarische Ökonomie Bremen')
+    parser.add_argument('-e', dest='e', type=str,
+                        help='Einkaufsrechnung einlesen')
+    parser.add_argument('-k', dest='k', type=str,
+                        help='Kontoauszug einlesen')
+    parser.add_argument('-i', dest='i', action='store_true',
+                        help='Show all items')
+    parser.add_argument('-b', dest='b', action='store_true',
+                        help='Process bank transactions')
+    parser.add_argument('-v', dest='v', action='store_true',
+                        help='Show version')
+    parser.set_defaults(i=False)
+    parser.set_defaults(p=False)
+    parser.set_defaults(b=False)
+    parser.add_argument('--server', dest='server', type=str,
+                        help='URL for API server')
+    parser.add_argument('--key', dest='key', type=str,
+                        help='API key')
+    parser.add_argument('--secret', dest='secret', type=str,
+                        help='API secrect')
+    parser.add_argument('---invoice-processor', dest='invoice_processor', type=str,
```

Comment:
there should be only two minus signs here.

Reply:
Fixed.